roles/ucp/templates/ucp.j2

```diff
@@ -49,17 +49,16 @@ start)
     /usr/bin/docker run --rm -t --name ucp \
         -v /var/run/docker.sock:/var/run/docker.sock \
         -e UCP_ADMIN_USER={{ ucp_admin_user }} -e UCP_ADMIN_PASSWORD={{ ucp_admin_password }} \
+    {% if run_as == "master" -%}
+        docker/ucp join {{ ucp_controller_replica }} --host-address={{ node_addr }} \
```

Comment:
@vvb interesting issue. Just curious, how did you validate that this was the issue? 

The order of command flags usually shouldn't  matter. Is this something to be reported to ucp folks?